.github/workflows/sqlx.yml

```diff
@@ -324,6 +324,16 @@ jobs:
           DATABASE_URL: postgres://postgres@localhost:5432/sqlx?sslmode=verify-ca&sslrootcert=.%2Ftests%2Fcerts%2Fca.crt&sslkey=.%2Ftests%2Fcerts%2Fkeys%2Fclient.key&sslcert=.%2Ftests%2Fcerts%2Fclient.crt
           RUSTFLAGS: --cfg postgres="${{ matrix.postgres }}"
 
+      # Direct SSL negotiation only supported by Postgres 17+
+      - if: matrix.postgres == '17'
```

Comment:
Does this work as `matrix.postgres >= '17'`? It's a string so it'd be lexicographical comparison but that _should_ still work.

Reply:
Looks like that works based on testing in my fork. Please see the latest push.

Reply:
9 is out of support so I wouldn't worry about that.

If it works, that's one less thing we have to remember to update.